code/src/components/SingleMoviePoster/SingleMoviePoster.css

```diff
@@ -0,0 +1,36 @@
+.moviePoster {
+    display: flex;
+    justify-content: center;
+    margin-top: 20%;
+}
+
+.moviePoster img {
+    width: 250px;
```

Comment:
Not important, but the image size could be dynamic (by using % and margin/padding to control the size) for a nicer user experience on the Single Movie page. Same with the plot text in tablet and desktop.